server/scsynth/CMakeLists.txt

```diff
@@ -38,10 +38,16 @@ if(NOT AUDIOAPI MATCHES "^(jack|coreaudio|portaudio|bela)$")
 endif()
 
 if(AUDIOAPI STREQUAL jack)
-  find_package(Jack)
-  if(NOT JACK_FOUND)
-    message(FATAL_ERROR "Jack selected as audio API, but development files not found")
+  # NOTE: this is a hack for Windows, which should rely on a weakref link against jack instead
```

Comment:
IMO the suggestion here is only one of possible solutions.
IIUC, the workaround (or hack, whatever) is provided to be able to build with jack _without relying on PkgConfig_. While regular linking might be suboptimal on Windows, it should technically work at least when building and running locally. And even if weak linking was used, we'd still need to provide jack headers, which might also need to be found somehow, if they are not provided with the library that provides weak linking.

This is not very important and the comment can stay as is. However, IMO, a more accurate note would state something like `this is a workaround for Windows if PkgConfig is not available`